stdlib/array.ml

```diff
@@ -303,7 +303,7 @@ let stable_sort cmp a =
 
 let fast_sort = stable_sort
 
-(** {6 Iterators} *)
+(** {1 Iterators} *)
```

Comment:
Not directly related to this PR, but there are two sections in `Array` called "Iterators" : the first one has `iter`, `map` and frieds; the second one is about `Seq.t`.  One should probably rename sections related to `Seq.t` to something like "Sequences".

Reply:
The reason why this is related to this PR is that before the PR, one could more easily grep for "6 Iterators".  So it would be (admittedly marginally) simpler to do the renaming as part or before this PR is merged.  @c-cube : do you agree with the suggested renaming?

Reply:
Personally I rather think that in a module `M` all functions that are for converting from one type to another (`M.{of,to}_{list,array,string,seq,...}`) should simply all be gathered under a `{1:conv Converting}` section and that folding/iterating functions should be gathered under a `{1:traversing Traversing}` section.

I would like at some point to make a (or multiple) PR to review and harmonize the documentation style and presentation order among the modules, but for now I fear a bit too much the bikesheding (maybe I'll come up with a description of the intended work first).

Reply:
Fair enough!